.eslintignore

```diff
@@ -5,4 +5,5 @@ cache/
 src/proto/youtube.ts
 coverage/
 node_modules/
-dist/
\ No newline at end of file
+dist/
+src/proto/generated/
```

Comment:
These files still seem to be commited. `git rm --cached src/proto/generated` should make git forget about the files.

Reply:
The generated protobuf stuff was commited previously too, so didn't change it
https://github.com/LuanRT/YouTube.js/blob/main/src/proto/youtube.ts